### PR TITLE
Removes Sec - Adds PMC

### PR DESCRIPTION
**Pull request by Stutternov**
## About The Pull Request
Since we are a server full of ideas and amazing forethought and totally no hypocrisy on the end of many people involved in its development / staff / 'ideas' - I will be standardizing all our ideas here.

Marshals and Blackshield have been **removed**!

They have been since replaced with Lonestar brand McMercenaries - since this server has a strange, quirky habit of fetishizing the idea of PMCs while sitting behind their computer.

This shall work as the Creative Director's approved Lonestar McMercs content **MERGED** with the Blackshield PMC rework promised to players for ages.

As such, here you go! I've added the asked for PR. It will be adjusted to be inline with the following McMercs path. Following the pathing:
- /mob/living/superior_animal/human/colony_allie/ship_breaker_marks
- /mob/living/superior_animal/human/colony_allie/ship_breaker_marks/sword

image
^ Will be redoing all the gear soon to look like this. ^

## "Rebel, is this a meme?"

No, honestly. It's not anymore. This is a PR for the braindead people who have asked for this, and the staff who just hypocritically DMed me to complain about code shit 'not fitting their lore idea' but approved this shit without much thought in their empty head.

## Changelog
:cl:
removes: Blackshield & Marshals
Adds: McChildkiller Mercenaries (Wagner Group is SS13)
/:cl:

### CODE REVIEW

File: code/datums/setup_option/backgrounds/origin_chtmant.dm [hunk 1]
```diff
@@ -45,15 +45,15 @@
 
 /datum/category_item/setup_option/background/ethnicity/chtmantra
 	name = "Ra Caste"
-	desc = "Ra are the warriors and sentries of the hives. Numbering in the hundreds they would tower over Ru�s and even \
+	desc = "Ra are the warriors and sentries of the hives. Numbering in the hundreds they would tower over Ru�s and even \
```
Comment (Guidesu) [inline code/datums/setup_option/backgrounds/origin_chtmant.dm:48]:
```suggestion
	desc = "Ra are the warriors and sentries of the hives. Numbering in the hundreds they would tower over Ru's and even \
```
Type

Comment (Guidesu) [reply code/datums/setup_option/backgrounds/origin_chtmant.dm:48]: *typo

**Comment by DorfGuy1** [comment]
holy shit rebel said 'fine you want mcmercs? here'

**Comment by Stutternov** [comment]
> holy shit rebel said 'fine you want mcmercs? here'

Cora Approved.
image

File: code/datums/setup_option/backgrounds/origin_chtmant.dm (hunk 1 above)
Comment (Stutternov) [reply code/datums/setup_option/backgrounds/origin_chtmant.dm]: thank u for the aid comrade, visual studio made the ' turn to that symbol.

**Comment by cdb-is-not-good** [comment]
Please no.

**Comment by Trilbyspaceclone** [comment]
Was requested to be closed

**Pull request closed without merging**